riotfile.py

```diff
@@ -1422,7 +1422,7 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
                 Venv(
                     pys=select_pys(min_version="3.8", max_version="3.11"),
                     pkgs={
-                        "aiobotocore": ["~=1.4.2", "~=2.0.0", latest],
+                        "aiobotocore": ["~=1.0.0", "~=1.4.2", "~=2.0.0", latest],
```

Comment:
What's the testing stragtegy here? Is it min, max and the latest release of every major version?

Reply:
min, max at the very least.